vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/registry/impl/InstallationScope.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jackrabbit.vault.packaging.registry.impl;
+
+import java.util.Arrays;
+
+public class InstallationScope {
```

Comment:
why not enum?

Reply:
@tripodsan  - see above, there is no way to access the values of the enum as a String with a static expression which I need for the annotations at build time.

Reply:
well, I think that having the benefit of the `enum` throughout the code outweighs to use 3 strings in the config :-)